### ISSUE TITLE
not work for dynamic added contents

### ISSUE THREAD

**Issue by moh85** [ryanwalters/sticky-events#15]
Hello,

the issue is that when a new content is added to page, the plugin will not work.
It only take the initial page height so it fails when a new content expand the document height.
any solutions?

**Comment by TotallWAR** [comment]
@moh85 Did you solve the problem? I hame the same one.

**Comment by jacksleight** [comment]
I been trying to get this working as well, as far as I can tell you need to completely remove and re-initialize StickyEvents when new content is added. Like this:

```js
const listener = (ev) => {
    console.log(ev.target, ev.detail);
};

let stickyInstance;

// Initialize
stickyInstance = new StickyEvents();
stickyInstance.stickyElements.forEach((sticky) => {
    sticky.addEventListener(StickyEvents.CHANGE, listener);
});

// --- dynamic content gets added ---

// Un-Initialize
stickyInstance.stickyElements.forEach((sticky) => {
    sticky.removeEventListener(StickyEvents.CHANGE, listener);
});
stickyInstance.disableEvents(true);

// Re-Initialize
stickyInstance = new StickyEvents();
stickyInstance.stickyElements.forEach((sticky) => {
    sticky.addEventListener(StickyEvents.CHANGE, listener);
});
```

**Comment by ryanwalters** [comment]
Hey guys, just released `3.1.0-beta.0` which adds support for adding stickies after initialization. If you guys wanted to test it out and let me know if it's doing what you expected, I'd appreciate the feedback!

```
npm install sticky-events@3.1.0-beta.0
```

You can see the updated README here:
https://github.com/ryanwalters/sticky-events/blob/addStickies/README.md

@moh85 @TotallWAR @jacksleight

**Comment by TotallWAR** [comment]
@ryanwalters it works, thanks!

**Comment by ryanwalters** [comment]
Great! This feature is included in the `sticky-events@3.1.0` release.
